examples/02-complex-example/generated/binding.ts

```diff
@@ -191,7 +191,9 @@ export interface UserWhereInput {
 }
 
 export interface UserWhereUniqueInput {
-  id?: ID_Input | null
+  id?: String | null
+  firstName?: String | null
```

Comment:
TODO: this is wrong.  Need to make sure `firstName` and `stringEnumField` are removed.